src/main/java/taxi/service/impl/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,34 @@
+package taxi.service.impl;
+
+import java.util.Optional;
+import taxi.exception.AuthenticationException;
+import taxi.lib.Inject;
+import taxi.lib.Service;
+import taxi.model.Driver;
+import taxi.service.AuthenticationService;
+import taxi.service.DriverService;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final String sqlInjectionPatten
+            = "\\b(ALTER|CREATE|DELETE|DROP|EXEC(UTE){0,1}|INSERT( +INTO){0,1}"
+            + "|MERGE|SELECT|UPDATE|UNION( +ALL){0,1})\\b";
+    @Inject
+    private DriverService driverService;
+
+    @Override
+    public Driver login(String login, String password) throws AuthenticationException {
+        if (password.matches(sqlInjectionPatten) | login.matches(sqlInjectionPatten)) {
+            throw new AuthenticationException("Nice try, contact us"
+                    + " if you succeed, pls");
+        }
```

Comment:
:smiling_face_with_tear:

Reply:
![image](https://user-images.githubusercontent.com/114599463/217224010-92602456-2d74-4978-937d-a46bde1e7ca8.png)
what i should do with this one?

Reply:
<img width="648" alt="Screenshot 2023-02-07 at 13 41 38" src="https://user-images.githubusercontent.com/29378916/217235703-58f5c80f-3039-4c1c-91ec-c46717a788d6.png">